net/FlatBuffers/Table.cs

```diff
@@ -83,15 +84,15 @@ protected int __vector(int offset)
         }
 
         // Initialize any Table-derived type to point to the union at the given offset.
-        protected TTable __union<TTable>(TTable t, int offset) where TTable : Table
+        public T __union<T>(int offset) where T : struct, Accessor
```

Comment:
As for Net Core, we should very definitely start looking at this - it's at 1.0 RTM now and is *the* cross platform version of .NET (Mac, Linux, Windows). It's a whole new compiler & runtime with a reduced library set - which you're right, we don't use - however there may be differences in performance. There's an AOT compiler (not sure if that's standard or optional), which will have a bearing (likely, positive!).

I've not used it much at all, so really can't say much more than that, unfortunately.